Makefile

```diff
@@ -25,14 +25,14 @@ help:
 	@echo "                   from Alexander Pozdnyakov which provides 4.1.0."
 	@echo "                   See https://launchpad.net/~alex-p/+archive/ubuntu/tesseract-ocr"
 	@echo "                   for details.)"
-	@echo "    deps          Install python deps via pip"
-	@echo "    deps-test     Install testing python deps via pip"
-	@echo "    install       Install"
+	@echo "    deps          Install Python deps for install via pip"
```

Comment:
If you add those descriptions as comments directly above the targets, you can run [`shinclude -i Makefile`](https://github.com/kba/shinclude) which will in turn (see the `BEGIN-EVAL` comment above) run [makefile-parser](https://github.com/kba/makefile-parser) to generate the help.

Reply:
Oh, I didn't know you were doing it like this!

And the order of the help comments is the order of the targets?

Reply:
So do you want me to rewrite those comments (and the order) of the rules?

Reply:
Yes, makefile-parser parses a subset of Makefile syntax (assignments and targets - does that remind you of somehting :-P) and pairs them with the immediately preceding comment and then outputs, in document order, help for targets and variables.

Reply:
It's up to you, but if you want to see how it is being generated by shinclude/makefile-parser, see https://github.com/OCR-D/ocrd_tesserocr/commit/c2f3ad84b3f47008a128afabd95cba71c116b4ac

Reply:
Wow, that's quite an undertaking! Does using regexps to partially parse the makefile allow you to check for unsupported syntax or traps, so you can warn the user?

What about `FIXME` comments and the like?

Reply:
> allow you to check for unsupported syntax or traps,

In a way yes, there will be warnings about `UNHANDLED LINE` if no regex matches (for many constructs, like conditionals and multiline variable-assignment.

> `FIXME` comments

I deleted those manually I have to admit :)

Reply:
9e6ad41 should suffice